// Test output.
void testPrint(const char * const str, ...);

exception ExampleEmptyError();

// Entry point.
int testEntryPoint(int argc, char ** argv) {
	try {
		throw ExampleEmptyError();
	} catch (ExampleEmptyError error) {
		testPrint("Caught empty exception!");
	}
	return 0;
}

